README.md

```diff
@@ -0,0 +1,158 @@
+# Safe Global Monorepo 🙋‍♂️
+
+## Overview
+
+Welcome to the Safe Global Monorepo! This repository houses multiple applications and packages managed under a unified
+structure using Yarn Workspaces. The monorepo setup simplifies dependency management and ensures consistent development
+practices across projects.
+
+### Key Components
+
+- **apps/**: Contains application projects (e.g., `mobile` for the Safe Mobile App).
+- **packages/**: Shared libraries and utilities.
+- **config/**: Configuration files for the monorepo.
+
+## Getting Started
+
+To get started, ensure you have the required tools installed and follow these steps:
+
+### Prerequisites
+
+- **Node.js**: Install the latest stable version from [Node.js](https://nodejs.org/).
+- **Yarn**: Use Yarn version 4.5.3 or later
+
+to install it with the latest node version you can simply do
+
+```bash
+corepack enable
+```
+
+and then just run
+
+```bash
+yarn
+```
+
+This will install the required version of yarn and resolve all dependencies.
+
+> [!INFO]
+>
+> Corepack is a tool to help with managing versions of your package managers. It exposes binary proxies for each supported package manager that, when called, will identify whatever package manager is
+> configured for the current project, download it if needed, and finally run it.
+
+### Initial Setup
+
+1. Clone the repository:
+
+```bash
+git clone <repo-url>
+cd monorepo
+```
+
+2. Install dependencies:
+
+```bash
+yarn install
+```
+
+## Monorepo Commands
+
+Here are some essential commands to help you navigate the monorepo:
+
+### Workspace Management
+
+- **Run a script in a specific workspace:**
+
+```bash
+yarn workspace <workspace-name> <script>
+```
+
+Example:
+
+```bash
+yarn workspace @safe-global/web start
+```
+
+- **Add a dependency to a specific workspace:**
+
+```bash
+yarn workspace <workspace-name> add <package-name>
```

Comment:
Is it not possible to cd into the app and run `yarn add <package-name>` there directly?

Reply:
No. When you execute the command it would run in the package directory context (meaning root will be the directory). Sometimes the output coloring is a bit off and yarn is adding some prefixes to every line (but I can't tell you when this exactly occurs)